sunpy/instr/rhessi.py

```diff
@@ -183,8 +183,9 @@ def hsi_linecolors():
     ----------
     `hsi_linecolors.pro <https://hesperia.gsfc.nasa.gov/ssw/hessi/idl/gen/hsi_linecolors.pro>`__
     """
-    return ('black', 'magenta', 'lime', 'cyan', 'y', 'red', 'blue', 'orange',
-            'olive')
+    return ('black', 'tab:pink', 'tab:green', 'tab:cyan',
```

Comment:
I think we should probably add this as a new cycle so people can opt back into the old one?

Reply:
Should I add it to `timeseries` to avoid adding anything new to `instr`? I think ideally any plot related things should be under `timeseries`, in a similar way to how the e.g. aia colormap is in `map` and not `instr`.

Reply:
yeah I am in favour of that.